connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -170,6 +233,24 @@ public void deleteConnector(String connName) throws IOException {
         }
     }
 
+    /**
+     * Get the connectors running in this cluster.
```

Comment:
Nit: this returns the _names_ of the connectors running in this cluster. Would be good to state this explicitly in the JavaDoc, including in the `@return` statement

Reply:
done